content/requests/sign_request_create_request.yml

```diff
@@ -0,0 +1,152 @@
+---
+title: Create a sign request
+
+type: object
+
+description: |-
+  A request to create a sign request object
+
+required:
+  - signers
+  - source_files
+
+properties:
+  is_document_preparation_needed:
+    type: boolean
+    description: >-
+      Indicates if the sender should be taken into the builder flow to
+      prepare the document, or if the document is already prepared.
+    example: true
+
+  redirect_url:
+    type: string
+    description: The uri that a signer will be redirect to after signing a
+      document. If no declined redirect url is specified, this will be
+      used for decline actions as well. 
+    example: https://redirect.com
+
+  declined_redirect_url:
+    type: string
+    description: >-
+      The uri that a signer will be redirect to after declining to sign a
+      document
+    example: https://declined-redirect.com
+
+  required_attachments:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+          example: passport
+          description: Name of file attachment you would like signers 
+            to upload
+    description: Attachments that signers are required to upload
+
+  are_attachments_disabled:
```

Comment:
I think changing this is fine for the `are_attachments_enabled` etc. (although all of these will need to default to true) but feels weird for the `are_emails_disabled` since emails will only really be disabled for a particular embed url flow -- I could update to something else like are_emails_suppressed?

Reply:
I think `suppressed` and `disabled` have the same "issue" of being "falsy".  If it doesn't make sense to use `are_emails_enabled`, then I'm fine with `are_emails_disabled`

Reply:
updated